frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/RolloutParameter/RolloutParameter.tsx

```diff
@@ -28,17 +28,19 @@ export const RolloutParameter: FC<{
     return (
         <>
             <StrategyEvaluationItem type='Rollout %'>
-                <StrategyEvaluationChip label={`${percentage}%`} /> of your base{' '}
-                {stickiness}
-                <span>
-                    {hasConstraints ? 'who match constraints ' : ' '}
-                    is included.
-                </span>
-                {displayGroupId && parameters?.groupId ? (
-                    <StrategyEvaluationChip
-                        label={`groupId: ${parameters?.groupId}`}
-                    />
-                ) : null}
+                <StrategyEvaluationChip label={`${percentage}%`} />
+                <div>
```

Comment:
can this be a `p` instead? It's text content, so it seems reasonable to me. Or in fact, if the chip is a span, then the entire child content (chip + rollout info) can (arguably should?) be in a single `p` tag.